### PR TITLE
Adjust supported python versions due to pytket version update

### PR DESCRIPTION
**Pull request by nquetschlich**

### CODE REVIEW

**Comment by nquetschlich** [comment]
pre-commit.ci autofix

File: pyproject.toml [hunk 1]
```diff
@@ -15,12 +15,12 @@ authors = [
...
 keywords = ["MQT",  "quantum computing", "benchmarking", "performance", "testing"]
 license = { file = "LICENSE" }
-requires-python = ">=3.8"
+requires-python = ">=3.9"
 dynamic = ["version"]
 
 dependencies = [
     "qiskit[all]>=0.39.2,<0.40.0",
-    "pytket>=1.9,<1.11",
+    "pytket>=1.11,<1.12",
```
Comment (burgholzer) [inline pyproject.toml:23]: I'd personally rather keep this change in the dependabot PR. To separate concerns a little.

**Review by burgholzer — APPROVED** [review]
Just a small nitpick. Otherwise this looks good. Many thanks 🙏

File: pyproject.toml (hunk 1 above)
Comment (nquetschlich) [reply pyproject.toml]: Done.

**Review by burgholzer — APPROVED** [review]

**Review by burgholzer — APPROVED** [review]

**Pull request merged**